modules/define.py

```diff
@@ -27,3 +45,36 @@ def define(self, event):
                 event["stderr"].write("No definitions found")
         else:
             event["stderr"].write("Failed to load results")
+
+    def random_word(self, event):
+        if not self.last_called or (time.time()-self.last_called >=
```

Comment:
This is a solid way of having a delay between calls but can you show me the part of the documentation that says we need to?